db/db_impl.cc

```diff
@@ -6539,9 +6539,35 @@ Status DBImpl::IngestExternalFile(
   // Cleanup
   ingestion_job.Cleanup(status);
 
+  if (status.ok()) {
```

Comment:
Here we trigger the event for all files at the end. Do we want to notify listener immediately after each file is ingested?